lib/jcsda-emc/spack-stack/stack/cmd/stack_cmds/create.py

```diff
@@ -162,6 +162,12 @@ def setup_env_parser(subparser):
                 modules (not the spack environment).""",
     )
 
+    subparser.add_argument(
+        "--modify-pkg",
+        action="append",
+        help="Modify selected package's and place in an environment-specific repository",
```

Comment:
To clarify the ability to specify multiple packages:
```suggestion
        help="Modify selected packages and place in an environment-specific repository",
```

Reply:
Should have been singular anyway. Fixed.